application/helpers/remotecontrol/remotecontrol_handle.php

```diff
@@ -1633,11 +1633,11 @@ public function delete_participants($sSessionKey, $iSurveyID, $aTokenIDs)
     * @access public
     * @param string $sSessionKey Auth credentials
     * @param int $iSurveyID Id of the Survey to get token properties
-    * @param int $iTokenID Id of the participant to check
+    * @param array|struct|int Array $aTokenQueryProperties of participant properties used to query the participant, or the token id as an integer 
```

Comment:
Is `struct` allowed in phpdoc? In PHP, array = hashtable = struct = array everything.